### ISSUE TITLE
Unclear text in the last paragraph of Section 2.3.5

### ISSUE THREAD

**Issue by aamelnikov** [ietf-wg-emailcore/emailcore#98]
Donald Eastlake <d3e3e3@gmail.com> wrote:

The first sentence of this paragraph is of questionable grammar
and, in any case, extremely hard to parse.

   When domain names are used in SMTP, and unless further restricted in
   this document, names that can be resolved to MX RRs or address (i.e.,
   A or AAAA) RRs (as discussed in Section 5) are permitted, as are
   CNAME RRs whose targets can be resolved, in turn, to MX or address
   RRs.

Assuming I have parsed it correctly, how about:

   Unless further restricted in this
   document, domain names used in SMTP are names that can be resolved
   to MX or address (A or AAAA) RRs (see also Section 5), or to CNAME
   RRs that can be resolved to an MX or address RR.

**Comment by aamelnikov** [comment]
Agreement during IETF 121 that this change should be made.

**Comment by klensin** [comment]
This change was implemented in draft-ietf-emailcore-rfc5321bis-34, posted 2024-11-07.
